payment-cards/pci-compliance/levels.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: PCI Levels
+permalink: /payment-cards/pci-compliance/levels/
+---
+
+Levels in the PCI world are based on the number of transactions you process anually.
+
+If you fall under levels 2, 3, or 4, you must fill out a [Self Assessment Questionnaire](/payment-cards/pci-compliance/saqs/).
```

Comment:
AOC and ASV scan? http://usa.visa.com/merchants/protect-your-business/cisp/merchant-pci-dss-compliance.jsp#anchor_4